### ISSUE TITLE
[Eclipse IDE] Accessibility: no mnemonic for new connection menu item

### ISSUE THREAD

**Issue by eharris369** [eclipse/codewind#2355]
For accessibility purposes, all menu items should have a mnemonic.  Add one for the new connection menu item.
